votes/initiateNewVote/_EDIT_ME.yml

```
@@ -35,7 +36,9 @@ canShuffleCandidates: true
 # 5. Insert here a short description of the vote objectives and link to the
 #    issue it was discussed on to give the full context.
 footerInstructions: |
-  TBD
+  The objective of the vote is to determine a basic question: do we continue bundling
+  corepack with Node.js or not, and if so, do we enable it by default or not. There
+  are additional questions to resolve that will be handled separately.
```

```suggestion
  The objective of the vote is to determine a basic question: do we continue bundling
  the corepack binary with Node.js or not, and if so, do we enable bundling jumper
  binaries for its supported package managers (such as yarn and pnpm) by default or not.
  There are additional questions to resolve that will be handled separately.
```

While I don't want to get into wordsmithing the options, I do believe that these instructions here can benefit of a more descriptive explanation of what we continue to bundle and what "enabled by default" means.